app/gui2/e2e/collapsingAndEntering.spec.ts

```diff
@@ -93,12 +94,16 @@ test('Collapsing nodes', async ({ page }) => {
 })
 
 async function expectInsideMain(page: Page) {
-  await expect(locate.graphNode(page)).toHaveCount(5)
+  await expect(locate.graphNode(page)).toHaveCount(9)
```

Comment:
Would it be possible to have this constant live with the source code where the initial nodes are defined?

Reply:
Well, again I hit the problem, that the source is defined in `mock/engine.ts` and test files does not clear access to it. I will skip straightening that up ATM